demos/insurance_agent/arch_config.yaml

```diff
@@ -26,6 +26,7 @@ endpoints:
 
 prompt_targets:
   - name: policy_qa
+    http_method: POST
```

Comment:
move in endpoint

Reply:
and update the reference spec in documentation.

Reply:
moved

Reply:
updated reference